src/library/simplified/extension/destination-city-name.js

```diff
@@ -69,15 +84,21 @@ else if (utag.isRailRateDetails() && b['entity.railSearch.railDetail.railLegs.0.
 {
     b['destination'] = b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.arrivalStation.name'];
 }
+else if (utag.isPPymt() && typeof b["entity.checkout.flightOffers.0.destinationAirportCityState"].split(",")[0] !== "undefined") {
```

Comment:
This will throw an error if b["entity.checkout.flightOffers.0.destinationAirportCityState"] is undefined